plugin/src/main/java/io/fabric8/maven/plugin/enricher/EnricherManager.java

```diff
@@ -135,9 +135,9 @@ public void enrich(ProcessorConfig config, KubernetesListBuilder builder) {
 
 
     private void logEnrichers(List<Enricher> enrichers) {
-        log.verbose("Enrichers:");
+        log.debug("Enrichers:");
```

Comment:
why do you choose `debug()` over `verbose()` ? Debug is very noisy and probably mostly switched off. The list of enrichers/generators though are more on the level of the build step of a Docker or S2I build which are also shown with verbose.

I probably was a bit misleading in my commetn with _is useful for debug_ ;-)